todoapp/projects/models.py

```
@@ -1,27 +1,32 @@
-    - Add unique constraints
-
-    Add string representation for this model with project name and user email/first name.
-    """
+
+    project = models.ForeignKey('Project', on_delete=models.CASCADE)
+    member = models.ForeignKey(settings.AUTH_USER_MODEL , on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ('project', 'member')
+
+    def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)